stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/sounds/RingingConfig.kt

```diff
@@ -83,21 +114,25 @@ public fun deviceRingtoneRingingConfig(context: Context): RingingConfig = object
  * @param context Context used for retrieving the sounds.
  * @param incomingCallSoundResId The resource ID for the incoming call sound.
  * @param outgoingCallSoundResId The resource ID for the outgoing call sound.
+ * @param playIncomingSoundIfMuted Whether to play the incoming sound even if the device is muted.
+ * @param playOutgoingSoundIfMuted Whether to play the outgoing sound even if the device is muted.
  */
 public fun resRingingConfig(
     context: Context,
     @RawRes incomingCallSoundResId: Int,
     @RawRes outgoingCallSoundResId: Int,
 ) = object : RingingConfig {
-    override val incomingCallSoundUri: Uri? = incomingCallSoundResId.toUriOrNUll(context)
-    override val outgoingCallSoundUri: Uri? = outgoingCallSoundResId.toUriOrNUll(context)
+    override val incomingCallSoundUri: Uri? = incomingCallSoundResId.toUriOrNull(context)
+    override val outgoingCallSoundUri: Uri? = outgoingCallSoundResId.toUriOrNull(context)
 }
 
 /**
  * Returns a ringing config that uses custom URIs for incoming and outgoing call sounds.
  *
  * @param incomingCallSoundUri The URI for the incoming call sound.
  * @param outgoingCallSoundUri The URI for the outgoing call sound.
+ * @param playIncomingSoundIfMuted Whether to play the incoming sound even if the device is muted.
```

Comment:
I think these 2 comments `playIncomingSoundIfMuted` `playOutgoingSoundIfMuted` should be removed

Reply:
@liviu-timar